Makefile

```
@@ -102,12 +102,17 @@ vet: ## Run go vet against code.
 ## --------------------------------------
 
 .PHONY: test
-test: manifests generate fmt vet local-unit-test
+test: manifests generate fmt vet local-unit-test integration-test
 
 .PHONY: local-unit-test
 local-unit-test: $(ENVTEST) ## Run tests.
 	CGO_ENABLED=1 KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test ./pkg/... -race -coverprofile=coverage.xml -covermode=atomic -v
 
+.PHONY: integration-test
+integration-test: $(ENVTEST) ## Run integration tests.
+	CGO_ENABLED=1 KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" \
+	go test ./test/apis/... -race -coverprofile=integration_coverage.xml -covermode=atomic
```

```suggestion
	ginkgo -v -p --race --cover --coverpkg=./... ./test/apis/...
```
please use the ginkgo -p so that we run these api tests in parallel.